hr-time/resources/unload_child.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<title>Helper page for ../unload-manual.html</title>
+<ol start="2">
+ <li><button id="navigation">Click here</button>
+ <li>Confirm the unload/reload dialog.
+</ol>
+<script>
+onload = function() {
+  opener.child_load_index++;
+
+  // after the reload
+  if (opener.child_load_index > 1) {
+    // child_origin is the expected time origin of the secondary window
+    var child_origin = opener.performance.now() - performance.now();
+    opener.t.step(function () {
+     opener.assert_approx_equals(opener.child_unload_time, child_origin, 10);
```

Comment:
No. It should be close to 0, as I understand it. 10s was unrelated (default testharness.js timeout, now disabled).